css/style.css

```diff
@@ -0,0 +1,43 @@
+#image {
+  max-width: 100%;
+  height: auto;
+  margin-top: 0px;
+  margin-right: auto;
+  margin-bottom: 0px;
+  margin-left: auto;
+}
+#img-div {
+  text-align: center;
+}
```

Comment:
I think you should leave a space between your CSS rules. This makes your CSS cleaner and more readable. I suggest you create a space between line 8 and line 9.